libeventheader-decode-cpp/src/EventFormatter.cpp

```diff
@@ -152,8 +319,9 @@ class StringBuilder
     }
 
     // Requires: there is room for utf8.size() chars.
+    // Assumes: utf8 is valid UTF-8.
     void
-    WriteUtf8(std::string_view utf8) noexcept
+    WriteUtf8Unchecked(std::string_view utf8) noexcept
```

Comment:
Note: Renamed `WriteUtf8` to `WriteUtf8Unchecked` to make it clear that the caller is asserting the value is valid UTF-8.